duniverse/mdx/.travis.yml

```diff
@@ -1,34 +1,19 @@
 language: c
+sudo: false
+services:
+  - docker
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
+script: bash ./.travis-docker.sh
```

Comment:
why did you change that file?

Reply:
This change and all the other comes are from the update of mdx, those are not done by me, but by updating mdx, I dragged them along